specs/fizzbuzz_spec.rb

```diff
@@ -0,0 +1,32 @@
+# write only unit tests for method fizzbuzz in class Solver
+
+require 'rspec'
+require '../solver'
+
+describe Solver do
+  describe '#fizzbuzz' do
+    context 'when number is divisible by 3' do
+      it 'returns fizz' do
+        expect(Solver.fizzbuzz(3)).to eq('fizz')
+      end
+    end
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/229285817-dc6fecb7-7959-4fc5-8fbd-d8b7ce8398e3.png)

- [ ] It would be great if you rename your **specs** folder to **spec** without `s` before when I run `rspec` command then `rspec` was not able to find any tests because by default it finds a folder with **spec** :+1:

Results :arrow_down:
![image](https://user-images.githubusercontent.com/93039359/229285926-ab52bc1a-8211-4029-b1ea-831bc158e36b.png)